cmd/ddev/cmd/logs.go

```diff
@@ -13,9 +14,10 @@ var (
 
 // DdevLogsCmd contains the "ddev logs" command
 var DdevLogsCmd = &cobra.Command{
-	Use:   "logs [projectname]",
-	Short: "Get the logs from your running services.",
-	Long:  `Uses 'docker logs' to display stdout from the running services.`,
+	ValidArgsFunction: ddevapp.GetProjectNamesFunc("all", 1),
```

Comment:
Should only allow running projects I think.

Reply:
Fixed